src/ui/numeralFormat.ts

```diff
@@ -187,19 +188,52 @@ class NumeralFormatter {
     return this.format(n, "0.00");
   }
 
+  parseCustomLargeNumber(str: string): number {
+    const numericRegExp = new RegExp('^(\-?\\d+\\.?\\d*)([' + extraNotations.join("") + ']?)$');
+    const match = str.match(numericRegExp);
+    if (match == null) {
+      return NaN;
+    }
+    const [, number, notation] = match;
+    const notationIndex = extraNotations.indexOf(notation);
+    if (notationIndex === -1) {
+      return NaN;
+    }
+    return parseFloat(number) * extraFormats[notationIndex];
+  }
+
+  furthestFrom0(n1: number, n2 = 0, n3 = 0): number {
```

Comment:
As we are parsing the same number 3 times, there can't be an instance where one is positive and the other is negative.
Even if there was, it would return the lowest number, which would be accurate, as we are working with a negative number in that case.

Reply:
What I meant is - this function doesn't check the input in any way and if you pass different numbers, then you can get results that are different from what the function's name says. So maybe a docstring or a different name for the function would be in order?

Reply:
This should be better then?

Reply:
https://github.com/danielyxie/bitburner/pull/3150/commits/1d07bf049ff9957831ceac21f5e5692aad86031c

Reply:
how about instead of `furthestFrom0` ``largestAbsoluteNumber`

Reply:
Renamed and pushed